modules/post/linux/gather/enum_protections.rb

```diff
@@ -60,93 +63,242 @@ def report(data)
+  def find_exes
+    apps = {
+      'aa-status' => 'AppArmor',
+      'aide' => 'Advanced Intrusion Detection Environment (AIDE)',
+      'apparmor' => 'AppArmor',
+      'auditd' => 'auditd',
+      'avast' => 'Avast',
+      'bastille' => 'Bastille',
+      'bulldog' => 'Bulldog',
+      'chkrootkit' => 'chkrootkit',
+      'clamav' => 'ClamAV',
+      'elastic-agent' => 'Elastic Security',
+      'firejail' => 'Firejail',
+      'firestarter' => 'Firestarted',
+      'fw-settings' => 'Uncomplicated FireWall (UFW)',
+      'getenforce' => 'SELinux',
+      'gradm' => 'grsecurity',
+      'gradm2' => 'grsecurity',
+      'honeyd' => 'Honeyd',
+      'iptables' => 'iptables',
+      'jailkit' => 'jailkit',
+      'logrotate' => 'logrotate',
+      'logwatch' => 'logwatch',
+      'lynis' => 'lynis',
+      'nagios' => 'nagios',
+      'oz-seccomp' => 'OZ',
+      'paxctl' => 'PaX',
+      'paxctld' => 'PaX',
+      'paxtest' => 'PaX',
+      'proxychains' => 'ProxyChains',
+      'psad' => 'psad',
+      'rkhunter' => 'rkhunter',
+      'snort' => 'snort',
+      'tcpdump' => 'tcpdump',
+      'thpot' => 'thpot',
+      'tiger' => 'tiger',
+      'tripwire' => 'tripwire',
+      'ufw' => 'Uncomplicated FireWall (UFW)',
+      'wireshark' => 'Wireshark'
+    }
+
+    apps.each do |app, appname|
       next unless command_exists? app
 
       path = cmd_exec "command -v #{app}"
       next unless path.start_with? '/'
 
       print_good "#{app} found: #{path}"
-      report path
+      report "#{appname}: #{path}"
+    end
+  end
+
+  def find_config
+    apps = {
+      '/bin/logrhythm' => 'LogRhythm Axon',
+      '/etc/aide/aide.conf' => 'Advanced Intrusion Detection Environment (AIDE)',
+      '/etc/chkrootkit' => 'chkrootkit [chkrootkit -q]',
+      '/etc/clamd.d/scan.conf' => 'ClamAV',
+      '/etc/fluent-bit' => 'Fluent Bit Log Collector',
+      '/etc/freshclam.conf' => 'ClamAV',
+      '/etc/init.d/avast' => 'Avast',
+      '/etc/init.d/avgd' => 'AVG',
+      '/etc/init.d/ds_agent' => 'Trend Micro Deep Instinct',
+      '/etc/init.d/fortisiem-linux-agent' => 'Fortinet FortiSIEM',
+      '/etc/init.d/kics' => 'Kaspersky Industrial CyberSecurity',
+      '/etc/init.d/limacharlie' => 'LimaCharlie Agent',
+      '/etc/init.d/qualys-cloud-agent' => 'Qualys EDR Cloud Agent',
+      '/etc/init.d/scsm' => 'LogRhythm System Monitor',
+      '/etc/init.d/sisamdagent' => 'Symantec EDR',
+      '/etc/init.d/splx' => 'Trend Micro Server Protect',
+      '/etc/init.d/threatconnect-envsvr' => 'ThreatConnect',
+      '/etc/logrhythm' => 'LogRhythm Axon',
+      '/etc/opt/f-secure' => 'WithSecure (F-Secure)',
+      '/etc/otelcol-sumo/sumologic.yaml' => 'Sumo Logic OTEL Collector',
+      '/etc/rkhunter.conf' => 'rkhunter',
+      '/etc/safedog/sdsvrd.conf' => 'Safedog',
+      '/etc/safedog/server/conf/sdsvrd.conf' => 'Safedog',
+      '/etc/tripwire' => 'TripWire',
+      '/opt/COMODO' => 'Comodo AV',
+      '/opt/CrowdStrike' => 'CrowdShite',
```

Comment:
CrowdShite?

Reply:
[Copy-pasta error](https://github.com/hackerschoice/hackshell/commit/35c7e7a6ec0f0c8cdfecd9f05fc3a706e6b1dbcc#diff-fb5cf3fef44f586b98b92ced0fe1ff11e7015465d029cde8254aa845b84d12faR901-R917), I sent #20093 to fix it.